docs/features.md

```diff
@@ -216,16 +216,16 @@ On the left side bar you can find other actions (from top to bottom):
 - ![](left_bar_actions/edit.png) **Open Machete File** &mdash; open the `machete` file in IntelliJ editor (to see what this editor can do see `Edit machete file` section)
 - ![](left_bar_actions/fetch.png) **Fetch All Remotes** &mdash; equivalent to `git fetch --all` command
 - The most suitable actions (each is equivalent to one of the context menu actions) for a current branch that include one or more of:
-    - ![](left_bar_actions/rebase.png) **Sync to Parent by Rebase...**
-    - ![](left_bar_actions/merge.png) **Sync to Parent by Merge**
+    - ![](left_bar_actions/rebase.png) **Sync Current to Parent by Rebase...**
+    - ![](left_bar_actions/merge.png) **Sync Current to Parent by Merge**
     - ![](left_bar_actions/push.png) **Push Current Branch**
     - ![](left_bar_actions/pull.png) **Pull Current Branch**
     - ![](left_bar_actions/reset.png) **Reset Current Branch to Remote**
     - ![](left_bar_actions/slideOut.png) **Slide Out Current Branch**
-    - ![](left_bar_actions/overrideForkPoint.png) **Override Fork Point of Current Branch**
+    - ![](left_bar_actions/overrideForkPoint.png) **Override Fork Point of Current...**
```

Comment:
I mean... it's not clear now _what current thing_ this one refers to.

In case of `Push/Pull` it'd be pretty obvious that it can only be about a branch... but here, with Fork Point being practically a Machete-specific term it'll be very unclear

Reply:
_Sync Current to Parent by {Merge,Rebase}_ also sounds awkward... I'd rather have _Sync to Parent by {Merge,Rebase}_, if not _Sync Current Branch to Parent by {Merge,Rebase}_ due to verbosity 🤔